test/parallel/test-cluster-inspector-debug-brk.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+// Flags: --inspect={PORT}
+// This test ensures that flag --inspect-brk and --debug-brk works properly
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
+
+const debuggerPort = common.PORT;
+const script = common.fixturesDir + '/empty.js';
+
+function fail() {
```

Comment:
Instead of this, please use `common.fail()` or `common.mustNotCall()`, which are designed to do essentially this same thing without cluttering up the tests.